### PR TITLE
[STRAT-83] Create tests for solar.py, cell.py, and solar_arry.py

### PR DESCRIPTION
**Pull request by Daniel-Q-Austin**
Updated tests for the modules in the Solar folder.

array.py had to be renamed to 'solar_array'. I think array is a reserved word and working with it was a hassle.

### CODE REVIEW

**Comment by Daniel-Q-Austin** [comment]
sorry for the 5 billion commits im still figuring out how to use github lol

File: solar/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (e-wai) [inline solar/.vscode/settings.json:1]: Could you remove this from your commit?

Comment (Daniel-Q-Austin) [reply solar/.vscode/settings.json]: Should be removed

**Comment by Daniel-Q-Austin** [comment]
Commits squashed and newlines added, I think it should be good.

**Comment by ufshk** [comment]
> Squash is nicely done - interestingly Git is still complaining about End of File newlines:
> image

This is despite the fact that there seems to be an empty line at the end of your files.

**Comment by ufshk** [comment]
I'm going to merge this anyway, because we haven't been consistent with EoF newlines.

**Pull request merged**